dspace-uclouvain/src/main/java/org/dspace/uclouvain/external/dilbert/client/DilbertClient.java

```diff
@@ -26,31 +28,70 @@ public DilbertClient(UCLouvainAuthorAuthorityAPIConfiguration uclouvainAuthority
     }
     
     /** 
-     * Search by student first name or/and second name and filtering by {filter} (ex: filter="student")
-     * It returns a list of student information
+     * Search by student first name or/and second name and filtering by {filter} (ex: filter="student").
+     * It returns a list of student information.
      * 
-     * @param term
-     * @param filter
-     * @return: A list of dial peron information 
+     * @param term: The name and/or second name of the student.
+     * @return: A list of dial person information.
      */
     public DialPerson[] getStudentByTermWithFilter(String term) {
         // GET {apiUrl}/searchAuthorDrupal.php?term={term}&filter={filter}
-        String filter = "&filter=" + uclouvainAuthorityConfiguration.getFilter();
-        String url = "/searchAuthorDrupal.php?term=" + term + filter;
+        String filter = "&filter=" + uclouvainAuthorityConfiguration.getAuthorFilter();
```

Comment:
Method name specify "Student" target but you use "author" filter config. It should be consistent